src/main/java/com/mastery/java/task/exceptions/EmployeeExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.mastery.java.task.exceptions;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class EmployeeExceptionHandler{
+
+    @ExceptionHandler(DuplicateEmployeeException.class)
+    public ResponseEntity<String> duplicateEmployeeHandler(DuplicateEmployeeException exception){
+        return ResponseEntity.ok(exception.getMessage());
+    }
+
+    @ExceptionHandler(InvalidDateException.class)
+    public ResponseEntity<String> invalidDateHandler(InvalidDateException exception){
+        return ResponseEntity.ok(exception.getMessage());
```

Comment:
do you going to add the default exception handler method?

Reply:
I didn't understand what to do

Reply:
At the moment you have 2 exception handlers for the specific exceptions, but what if you got some unxepected exception, as example you DB will crashed? I suggest to add exception handler for the Exception.class just to show users some generic error page in case of some unexpected error.

Reply:
i added 404 error page